concent_api/core/models.py

```diff
@@ -44,11 +61,11 @@ class Meta:
         verbose_name_plural = ('ReceiveOutOfBand statuses')
 
 
-class MessageAuth(models.Model):
+class MessageAuth(Model):
     """
     This class is used to store provider and requestor keys for message exchange related to given initial message.
     """
 
-    message              = models.OneToOneField('Message', related_name='auth')
+    message              = OneToOneField('Message', related_name='auth')
```

Comment:
Same here. This change should be in the same fixup with the change above.

EDIT: To be clear, I don't mean a standalone commit marked as fixup. I mean that the original commit that adds the models should be amended to already have this change.